lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -3921,6 +3921,28 @@ def model_find_and_add_construction(model, climate_zone_set, intended_surface_ty
                                 'building_category' => building_category,
                                 'minimum_percent_of_surface' => minimum_percent_of_surface,
                                 'maximum_percent_of_surface' => maximum_percent_of_surface)
+
+      # 12/14/2021 added to test data pull from sub climate zones for transparent surfaces
+      if !props
+        if climate_zone_set == "ClimateZone 3"
+          # for PRM, climate zone 3 should test on 3A, 3B and 3C for transparent surfaces
+          search_test_set = ["A", "B", "C"]
+          for test in search_test_set
+            props = model_find_object(standards_data['construction_properties'],
+                                      'template' => template,
+                                      'climate_zone_set' => climate_zone_set+test,
+                                      'intended_surface_type' => intended_surface_type,
+                                      'standards_construction_type' => standards_construction_type,
+                                      'building_category' => building_category,
+                                      'minimum_percent_of_surface' => minimum_percent_of_surface,
+                                      'maximum_percent_of_surface' => maximum_percent_of_surface)
+            if props
+              # Found the transparent surfaces, stop the searching.
+              break
+            end
```

Comment:
Unless I'm missing something I think that there is a potential issue with the proposed changes. For example, if the targeted climate zone is 3B and a specific code version (or `template`) has different requirements for 3A and 3B then it seems that the proposed changes would return the requirement for climate zone 3A and not 3B. We might not currently have different requirements for 3A and 3B for the PRM but because the proposed changes are in `Standards.Model` which is used by other projects within openstudio-standards it might be an issue at some point.

Reply:
That is correct. However, the specific sub climate zone info is lost after [model_get_climate_zone_set_from_list ](https://github.com/NREL/openstudio-standards/blob/9b2e994bd77651482e0a444cb2f17009b941c468/lib/openstudio-standards/standards/Standards.Model.rb#L4292) . 

The scenario is possible, One modification I could suggest is not to call the model_get_climate_zone_set_from_list function. Instead: 

For example:
props = model_find_object(standards_data['construction_properties'],
                                'template' => template,
                                'climate_zone_set' => climate_zone_set.min,
                                'intended_surface_type' => intended_surface_type,
                                'standards_construction_type' => standards_construction_type,
                                'building_category' => building_category)

if !prop:
props = model_find_object(standards_data['construction_properties'],
                                'template' => template,
                                'climate_zone_set' => climate_zone_set.max,
                                'intended_surface_type' => intended_surface_type,
                                'standards_construction_type' => standards_construction_type,
                                'building_category' => building_category)